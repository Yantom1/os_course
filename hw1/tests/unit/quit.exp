smash> smash> smash> /tmp/smash_test
smash> smash pid is \d+
smash> smash: got ctrl-Z
smash: process \d+ was stopped
smash> \[1\] sleep 100& : \d+ \d+ secs
\[2\] sleep 50& : \d+ \d+ secs
\[3\] sleep 20 : \d+ \d+ secs \(stopped\)
smash> smash: sending SIGKILL signal to 3 jobs:
\d+: sleep 100&
\d+: sleep 50&
\d+: sleep 20
