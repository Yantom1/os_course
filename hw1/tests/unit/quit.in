sleep 100&
sleep 50&
pwd
showpid&
sleep 20
CtrlZ
jobs
quit kill
