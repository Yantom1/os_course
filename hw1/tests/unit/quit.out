smash> smash> smash> /tmp/smash_test
smash> smash pid is 44811
smash> smash: got ctrl-Z
smash: process 44819 was stopped
smash> [1] sleep 100& : 44813 1 secs
[2] sleep 50& : 44815 1 secs
[3] sleep 20 : 44819 1 secs (stopped)
smash> smash: sending SIGKILL signal to 3 jobs:
44813: sleep 100&
44815: sleep 50&
44819: sleep 20
